### PR TITLE
tvmosaic: initial addon

### PR DESCRIPTION
**Pull request by awiouy**

### CODE REVIEW

File: packages/addons/service/tvmosaic/package.mk [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+PKG_BUILD_FLAGS="-strip"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_NAME="TV Mosaic"
+PKG_ADDON_TYPE="xbmc.service"
+PKG_MAINTAINER="Anton Voyl (awiouy)"
+
+addon() {
+  mkdir -p $ADDON_BUILD/$PKG_ADDON_ID/bin
+  cp $PKG_BUILD/tvmosaic/tvmosaic_reg \
```
Comment (chewitt) [inline packages/addons/service/tvmosaic/package.mk]: minor nit.. traditionally we further indent `cp` under the `mkdir` that contains the files being copied

Comment (awiouy) [reply packages/addons/service/tvmosaic/package.mk]: Thanks. Done.

Comment (vpeter4) [reply packages/addons/service/tvmosaic/package.mk]: Indent with two spaces not only one :)

Comment (awiouy) [reply packages/addons/service/tvmosaic/package.mk]: Thanks. Done.

**Pull request merged**